bzt/modules/pbench.py

```diff
@@ -143,7 +143,7 @@ def __init__(self, executor, base_logger):
         self.port = 80
         self._target = {"scheme": None, "netloc": None}
 
-    def generate_config(self, scenario, load, hostaliases):
+    def generate_config(self, scenario, load, hostaliases=()):
```

Comment:
I kept it for possible future uses, since it is not hostaliases mechanism, rather "host to IP mapping"